rosdep/python.yaml

```diff
@@ -8159,6 +8159,24 @@ python3-pyqrcode:
   gentoo: [dev-python/pyqrcode]
   nixos: [python3Packages.pyqrcode]
   ubuntu: [python3-pyqrcode]
+python3-pyqt5:
+  alpine: [py3-qt5]
+  arch: [python-pyqt5]
+  debian:
+    '*': [pyqt5-dev, python3-pyqt5, python3-pyqt5.qtsvg, python3-sip-dev, qtbase5-dev]
```

Comment:
Why are you using the wildcard notation and calling out jessie and stretch? They appear to be the same keys.

Reply:
This was a copy paste artifact. it is now removed.